apps/caliper.cc

```diff
@@ -8,7 +8,12 @@
    BSD-3 License can be found in the LICENSE file of the repository.
 */
 
-/*
+/**
+  /file caliper.cc
+
+  /brief Add fictitious function calls at entry and exit point of each external
```

Comment:
I'll have to see about adding the apps directory to the Doxygen generation: right now, it doesn't show up when the doxygen target is built.   Does Doxygen recognize forward slashes as commands?  I'm only familiar with backslashes.

Reply:
I think you're right about Doxygen and the backslashes. Let me fix that and add the mark_executable demo and resubmit

Reply:
Should be fixed in 9620f28